### PR TITLE
Fix no ::YAML::ENGINE

### PR DESCRIPTION
**Pull request by y00rb**
in 2.2.1, doesn't have ::YAML::ENGINE.
::YAML is Psych class. so adding in condition.

### CODE REVIEW

File: lib/paper_trail/serializers/yaml.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def where_object_condition(arel_field, field, value)
       # in the serialized object_changes
       def where_object_changes_condition(arel_field, field, value)
         # Need to check first (before) and secondary (after) fields
-        if defined?(::YAML::ENGINE) && ::YAML::ENGINE.yamler == 'psych'
+        if defined?(::YAML::ENGINE) && ::YAML::ENGINE.yamler == 'psych' || ::YAML == Psych
```
Comment (batter) [inline lib/paper_trail/serializers/yaml.rb]: Can you qualify Psych with `::Psych`? I think that would make the Travis errors stop failing.

**Comment by y00rb** [comment]
@batter What's ruby version setting in Travis?

**Comment by y00rb** [comment]
@batter  fixed

**Comment by batter** [comment]
Good call, thanks for the PR!

**Pull request merged**